setup.cfg

```diff
@@ -50,7 +50,7 @@ install_requires =
     pycryptodomex  # for EncryptedKeyring backend in keyrings.alt
     pydantic >= 1.9.0
     pynwb >= 1.0.3,!=1.1.0
-    nwbinspector >= 0.3.1
+    nwbinspector @ git+https://github.com/NeurodataWithoutBorders/nwbinspector@dev
```

Comment:
note that for PR to get merged we better have a released version.

Moreover: filed https://github.com/NeurodataWithoutBorders/nwbinspector/issues/116 , attn @jwodder .

Reply:
Yep, that's no problem, just wanted to make sure it works for now.